openlibrary/macros/YouTube.html

```diff
@@ -6,8 +6,8 @@
 
 $if id in allowed_videos:
     <object width="640" height="385">
-        <param name="movie" value="http://www.youtube.com/v/$id&hl=en_US&fs=1&"></param>
-        <param name="allowFullScreen" value="true"></param>
-        <param name="allowscriptaccess" value="always"></param>
-        <embed src="http://www.youtube.com/v/$id&hl=en_US&fs=1&" type="application/x-shockwave-flash" allowscriptaccess="always" allowfullscreen="true" width="640" height="385"></embed>
+        <param name="movie" value="http://www.youtube.com/v/$id&hl=en_US&fs=1&" />
```

Comment:
The previous setup was valid HTML. Why is this better? Was this an intended change?

Reply:
param and embed tags give an warning when they are closed 
https://www.w3schools.com/tags/tag_param.asp
that's why I used this